apps/sms/test/marionette/lib/messages.js

```diff
@@ -9,6 +9,26 @@
     BACKSPACE: '\ue003'
   };
 
+  function observeElementStability(el) {
+    delete el.dataset.__stable;
+
+    function markElementAsStable() {
+      return setTimeout(function() {
+        el.dataset.__stable = 'true';
+        observer.disconnect();
+      }, 1000);
```

Comment:
> ya I know there might be several changes, but doesn't each change trigger the observer callback? And each callback will reset the timeout again. 

Yeah that's right, so the idea is to "debounce" marking element as stable. That's how I think of it - If change is occured, let's cancel (clearTimeout) previous intention to mark element as stable and wait for 1 sec one more time, if change is occurred once again (ex. after 900 ms) - that means the element isn't stable enough, so there is a chance that it will be changed again, so we cancel old timeout (~100ms left to mark element as stable) and again wait for 1sec.

So depending on the environment "speed" and length of string that we send with ```sendKeys``` there can happen different number of mutations just for one "addRecipient". Also there's a chance that MutationObserver call won't fire at all, that's why I call ```markElementAsStable``` at the very beginning.